packages/ember-glimmer/tests/integration/helpers/closure-action-test.js

```diff
@@ -246,40 +247,27 @@ moduleFor('Helpers test: closure {{action}}', class extends RenderingTest {
   }
 
   ['@test an error is triggered when bound action function is undefined']() {
-    let InnerComponent = Component.extend({
-    });
-
-    let OuterComponent = Component.extend({
+    this.registerComponent('inner-component', {
+      template: 'inner'
     });
-
-    this.registerComponent('inner-component', { ComponentClass: InnerComponent, template: 'inner' });
     this.registerComponent('outer-component', {
-      ComponentClass: OuterComponent,
       template: '{{inner-component submit=(action somethingThatIsUndefined)}}'
     });
 
-    // TODO: Improve this to test that the name of the parameter is contained
-    // within the error message.
     this.assert.throws(() => {
       this.render('{{outer-component}}');
     }, /An action could not be made for `.*` in .*\. Please confirm that you are using either a quoted action name \(i\.e\. `\(action '.*'\)`\) or a function available in .*\./);
```

Comment:
I actually think this should assert for the same error as the test below it: looks like it should throw a compile-time error than a run-time error. (which actually means we don't really have a test for compile-time check for UNDEFINED_REFERNECE or NULL_REFERENCE.